src/vs/workbench/parts/extensions/electron-browser/media/extensionEditor.css

```diff
@@ -312,8 +313,9 @@
 }
 
 .extension-editor .subcontent .monaco-tree-row .dependency > .icon {
-	height: 40px;
-	width: 40px;
+	height: 60px;
+	width: 60px;
+	object-fit: contain;
```

Comment:
I agree with the need for `object-fit`
The change in the height and width will make the image bigger and make it look like there is no padding/margin between various dependencies

<img width="519" alt="screen shot 2018-06-23 at 12 30 54 pm" src="https://user-images.githubusercontent.com/16890566/41812961-538f4e5a-76e1-11e8-9a90-4e6206b37bb8.png">

In your particular example of azure account, I would suggest to log an issue in the repository for that extension, to use a better image